MEEW-Client/MEEW-Client/Screens/ToDo/Controller/ToDoVC.swift

```diff
@@ -8,143 +8,124 @@
 import UIKit
 
 class ToDoVC: BaseVC {
-    
-    // MARK: - var & let
-    
+  
+  // MARK: - @IBOutlet
+  @IBOutlet weak var questionLabel: UILabel!
+  @IBOutlet weak var originalButton: UIButton!
+  @IBOutlet weak var archiveButton: UIButton!
+  @IBOutlet weak var inforButton: UIButton!
+  @IBOutlet weak var doneButton: UIButton!
+  @IBOutlet weak var imageView: UIImageView!
+  @IBOutlet weak var checkBoxButton1: UIButton!
+  @IBOutlet weak var checkBoxButton2: UIButton!
+  @IBOutlet weak var checkBoxButton3: UIButton!
+  @IBOutlet weak var checkBoxButton4: UIButton!
+  @IBOutlet weak var missionLabel1: UILabel!
+  @IBOutlet weak var missionLabel2: UILabel!
+  @IBOutlet weak var missionLabel3: UILabel!
+  @IBOutlet weak var missionLabel4: UILabel!
+  
+  // MARK: - @IBAction
+  @IBAction func touchUpToGoResult(_ sender: Any) {
+    guard let vc = UIStoryboard(name: "ResultVC", bundle: nil).instantiateViewController(withIdentifier: "ResultVC") as? ResultVC else { return }
+    vc.modalPresentationStyle = .fullScreen
+    present(vc, animated: true, completion: nil)
+  }
+  
+  @IBAction func againButton(_ sender: Any) {
+    viewWillAppear(true)
+  }
+  
+  @IBAction func inforButtonTapped(_ sender: Any) {
+    if let name = inforButton.titleLabel?.text {
+      MEEWPopUp.loadFromXib()
+        .setTitle(name)
+        .setDescription("\(name)로 살아볼까요?")
+        .present()
+    }
+  }
+  
+  @IBAction func originalButtonTapped(_ sender: Any) {
+    /// 수정하기
+    guard let vc = UIStoryboard(name: "ToDoVC", bundle: nil).instantiateViewController(withIdentifier: "VCVC") as? VCVC else { return }
+    vc.modalPresentationStyle = .fullScreen
+    vc.modalPresentationStyle = .overFullScreen
+    vc.modalTransitionStyle = .crossDissolve
+    present(vc, animated: true, completion: nil)
+  }
+  
+  @IBAction func checkBox1Tapped(_ sender: UIButton) {
+    sender.isSelected.toggle()
+  }
+  @IBAction func checkBox2Tapped(_ sender: UIButton) {
+    sender.isSelected.toggle()
+  }
+  @IBAction func checkBox3Tapped(_ sender: UIButton) {
+    sender.isSelected.toggle()
+  }
+  @IBAction func checkBox4Tapped(_ sender: UIButton) {
+    sender.isSelected.toggle()
+  }
```

Comment:
```suggestion
  @IBAction func checkBoxTapped(_ sender: UIButton) {
    sender.isSelected.toggle()
  }
```

이 코드처럼 4개의 체크박스를 각각 하나의 메서드에 모두 연결 지어줄 수 있어요.
그래도 동작은 이전 코드와 동일하게 실행이 됩니다.

![스크린샷 2022-02-25 오후 6 23 35](https://user-images.githubusercontent.com/61109660/155690028-626ce025-54f7-485c-8ef4-b3e1a5302538.png)

그럴 수 있는 이유는 인자로 넘어오는 sender에서 찾아볼 수 있는데요.
sender란 메서드를 호출하고 있는 개체 자체를 의미합니다.

어떤 버튼을 누르고 있는지 애초에 구분이 되고 있는거죠.

checkBoxTapped라는 메서드를 
첫번째 체크박스 버튼이 호출한건지,
두번째 체크박스 버튼이 호출한건지,
구분이 되고 있고 그것은 sender를 통해서 알 수 있습니다.

https://developer.apple.com/documentation/uikit/uiaction/3600315-sender

Reply:
고민이였는데 사실 가능한거였군요..예리한 센스 감사합니다👍👍